github-actions/previews/upload-artifacts-to-firebase/action.yml

```diff
@@ -89,7 +89,7 @@ runs:
         expires: 10d
         projectId: '${{inputs.firebase-project-id}}'
         entryPoint: '${{inputs.firebase-config-dir}}'
-        channelId: pr-${{github.repository}}-${{steps.artifact-info.outputs.unsafe-pull-number}}
+        channelId: pr-${{inputs.workflow-artifact-name}}-${{github.repository}}-${{steps.artifact-info.outputs.unsafe-pull-number}}
```

Comment:
iirc there is a length limit and it will truncate the PR from the visible url. so might be worth swapping order

Reply:
I am not sure I understand what swap you are meaning.  Happy to reorder the elements here, just not sure

Reply:
Just to capture the thougt:

Firebase truncates the URL that will be given to the user. This may cut off the PR number and make the preview URLs rather meaningless. Hence I propose swapping the PR number with the artifact name so that the PR number is more likely to be included. The full URL is still used internally in Firebase to identify the preview channel (different to the truncated URL derived from the id)